src/Admin/BaseBlockAdmin.php

```diff
@@ -102,7 +109,14 @@ public function preUpdate($object)
      */
     public function postUpdate($object)
     {
-        $this->blockManager->get($object)->postUpdate($object);
+        if (\is_callable([$object, 'postUpdate'])) {
+            $this->blockManager->get($object)->postUpdate($object);
+
+            @trigger_error(
+                'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in sonata/block-bundle 4.0.',
```

Comment:
```suggestion
                'The '.__METHOD__.'() method is deprecated since sonata-project/block-bundle 3.x and will be removed in version 4.0.',
```